### ISSUE TITLE
Just a comment...  keydiscovery and tracking?

### ISSUE THREAD

**Issue by magicalo** [tfpauly/draft-pauly-adaptive-dns-privacy#103]
Is it still possible to "track" the user by issuing a per-requestor public key from the Target server?  I presume a keydiscovery is done only once by a client (or infrequently enough that it can provide a tracking-vector).

If the above is accurate, would it be useful for a client to detect this case by doing keydiscovery more than once -- and/or -- have the client request this on some randomized basis to thwart such tracking techniques.

If I am totally misunderstanding this, I apologize in advance.

If I understood the role of the proxy correctly - the Proxy is terminating the DoH session and creating a new session to the Target.  This ensures IP privacy and crypto-privacy.  Correct?

**Comment by tfpauly** [comment]
The key is distributed in DNS records, like the TLS client hello key. Generally, the entity that serves up these DNS records to clients is a recursive that is entirely separate from the record owner, and the record is cached. Targeting one client shouldn’t be feasible. This is even harder to do with DNSSEC.

**Comment by tfpauly** [comment]
Sounds like mainly this can just have more explanation/discussion

**Comment by chris-wood** [comment]
@tfpauly since discovery is now out of scope, I recommend we close this without action.

**Comment by tfpauly** [comment]
Agreed
